test/Microsoft.NET.Build.Tests/GivenThereAreDefaultItems.cs

```diff
@@ -370,6 +370,13 @@ public void Default_items_have_the_correct_relative_paths()
         [Fact]
         public void Compile_items_can_be_explicitly_specified_while_default_EmbeddedResource_items_are_used()
         {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
Same here, would much rather this be dynamically skipped than falsely marking it as passing